code/datums/traits.dm

```diff
@@ -191,13 +185,24 @@
 	icon_state = "placeholder"
 	var/id = ""        //Unique ID
 	var/points = 0	   //The change in points when this is selected.
-	var/category = null //If set to a non-null string, People will only be able to pick one trait of any given category
+	var/list/category = null //If set to a non-null string, People will only be able to pick one trait of any given category
 	var/unselectable = FALSE //If TRUE, trait can not be select at char setup
 	var/requiredUnlock = null //If set to a string, the xp unlock of that name is required for this to be selectable.
 	var/cleanName = ""   //Name without any additional information.
 	var/isMoveTrait = FALSE // If TRUE, onMove will be called each movement step from the holder's mob
 	var/datum/mutantrace/mutantRace = null //If set, should be in the "species" category.
 
+	New()
+		..()
+
+		// Clean name should always be set, but if not, this replaces the (...) and [...]
+		// from the name and sets that as the clean name. Should probably just remove this and move cleanName to name as normal name never gets used anymore.
```

Comment:
> Should probably just remove this and move cleanName to name as normal name never gets used anymore.

:)

Reply:
This PR is already big enough to have been sitting here for a couple of weeks now.

Reply:
I can open a draft PR for after this one later to replace all of these

Reply:
https://github.com/goonstation/goonstation/pull/9919